samples/news/News.UITests/Tests.cs

```diff
@@ -40,18 +40,21 @@ public void SearchForMicrosoftNews()
         }
 
         [Test]
-        [Ignore] //Remove ignore tag after client secrets are implemented on DevOps
         public void AddAndRemoveFavorite()
         {
             var homePage = new HomePage();
```

Comment:
Remove this variable to store homepage, this is not the style of POP UITests. This should look something like :

```
new HomePage()
        .ShowNewsTab()
        .AddFavorite();
        .ShowNewsTab()
        .AddFavorite();

new  HomePage()
            .ShowFavoritesTab()
            .RemoveFavorite();
            .ShowFavoritesTab()
            .WaitToBecomeNotEmpty()
            .RemoveFavorite()
            .WaitToBecomeNotEmpty()
            .RefreshFavorites()
            .WaitToBecomeEmpty();
```